iiblib/iib_client.py

```diff
@@ -404,3 +405,78 @@ def rebuild_index(self, index_image):
 
     def health(self):
         raise NotImplementedError
+
+    def add_deprecations(
+        self,
+        index_image=None,
+        deprecation_schema=None,
+        operator_package=None,
+        binary_image=None,
+        build_tags=None,
+        overwrite_from_index=False,
+        overwrite_from_index_token=None,
+        raw=False,
+    ):
+        """Add deprecations of an operator package to index image.
+
+        Args:
+            index_image (str)
+                Index image ref used as source to rebuild
+            deprecation_schema (str)
+                Link of deprecation schema
```

Comment:
@emilyzheng How do you expect to submit deprecation_schema? IIB will accept a json string in the deprecation_schema

Reply:
It can submit any string. A json string would be like '{"a": "b"}'. The description was copied from https://github.com/release-engineering/iiblib/blob/master/iiblib/iib_build_details_model.py#L539. Seems `link` is not accurate? It's updated in the new patch.